src/utils/debounce.js

```diff
@@ -0,0 +1,7 @@
+export default debounce = (fn, duration) => {
```

Comment:
Maybe use debounce from lodash which we already include in the project (one function at a time)?